README.md

```diff
@@ -44,3 +45,121 @@ Ensure that you have the correct path by looking for the file `Beyond-All-Reason
 6. If developing Chobby also clone the code into the `games` directory. Follow the guide in the [Chobby README](https://github.com/beyond-all-reason/BYAR-Chobby#developing-the-lobby).
 
 More on the `.sdd` directory to run raw LUA and the structure expected by Spring Engine is [documented here](https://springrts.com/wiki/Gamedev:Structure).
+
+---
+
+## Automated Testing
+
+### Prereqs
+
+**Lua 5.1**
+
+_debian/linux_
+
+```zsh
+sudo apt install -y lua5.1
+```
+
+_windows_ (MSYS2 UCRT64)
+
+```zsh
+pacman -S --needed mingw-w64-ucrt-x86_64-lua51
+```
+
+_macOS_
+
+```zsh
+brew install lua@5.1
+```
+
+**Lux Package Manager**
+Follow the [Lux Getting Started Guide](https://lux.lumen-labs.org/tutorial/getting-started/).
+
+Or follow the Cargo instructions to manually build [on the Lux Github](https://github.com/lumen-oss/lux?tab=readme-ov-file#wrench-building-from-source)
+
+### Install Project Packages
+
+From the repo root (where `lux.toml` lives):
+
+```zsh
+lux --max-jobs=2 update
+```
+Note: in my testing `--max-jobs` was super specific to my machine and anything above that number would sometimes cause deadlocks.
+
+
+### Running Tests
+
+Run the full suite (via [Busted](https://lunarmodules.github.io/busted/)):
+
+```zsh
+# preferred for predictable CLI behavior
+busted
+```
+
+Filter by tag:
+
+```zsh
+busted -t focus
+```
+
+Optionally, run through Lux’s wrapper:
+
+```zsh
+lx test
+# run the emmylua type check
+lx check
+# or to drop into a shell so you can run `busted` manually
+lx shell --test
+busted
+8 successes / 0 failures / 0 errors / 0 pending : 0.246881 seconds
+```
+
+See Lux [Guides](https://lux.lumen-labs.org/guides/formatting-linting) for more information.
+
+Inspect objects inline while debugging:
+
+```lua
+print(require("inspect")(someObject))
+```
+
+### Notes
+
+- In specs, use `require("relative/path/to/file")`.
+  `VFS.Include` is aliased to `require` in `spec/spec_helper.lua`, but `require` makes it clear you’re referencing test code.
```

Comment:
@Ruwetuin I'm honestly unsure if this is adding any value, but it's what I'd been doing so figured I'd write it down and see what yall thought.